### ISSUE TITLE
调用registerApp方法崩溃

### ISSUE THREAD

**Issue by Shaw003** [sinaweibosdk/weibo_ios_sdk#434]
您好，我是一名iOS开发者，通过手动方式导入的WeiboSDK，我将接入微博的功能写在了一个framework工程中，并在主工程中导入了这个framework的子工程，目前出现了调用WeiboSDK的registerApp方法时，崩溃的问题，在模拟器上必现，我看了贵司提供的手册，手册上提到了使用UIPastborad导致主线程卡死问题，但是我的工程中并没有使用该API，并且我也按照提示，尝试将registerApp放到了子线程中，依然崩溃，XCode提示崩溃原因为Thread 1: EXC_BAD_INSTRUCTION (code=EXC_I386_INVOP, subcode=0x0)，请求帮助解决，谢谢
以下是我的开发信息：
Xcode版本：Version 11.3.1 (11C504)
WeiboSDK版本：3.2.7
iOS开发语言版本：framework子工程：Swift5
                                主工程：Swift4
模拟器型号版本：iPhone8Plus 11.0.1，无论那个模拟器都必现崩溃

**Comment by Shaw003** [comment]
SDK版本：003233000

**Comment by MengLiMing** [comment]
我出现了相同的问题，也是封装基础模块framework，分享功能，手动微博SDK 崩溃

**Comment by MengLiMing** [comment]
猜测可能是因为找不到bundle，现将bundle放入主工程，可以运行

**Comment by mihailogazda** [comment]
Yes without the bundle registerApp will try to assign a public certificate that is part of the bundle, and crash when its not included. Hope that helps.
